server/templates/place.html

```diff
@@ -17,21 +17,26 @@
 
 {% set main_id = 'dc-places' %}
 {% set page_id = 'page-dc-places' %}
-{% set title = place_name + ' ' + category + ' - ' + _('Place Explorer') %}
+{% if category == '' %}
+  {% set title = place_name + ' - ' + _('Place Explorer') %}
+  {% set description = 'Statistics about economics, health, equity, crime, education, demographics, housing, and environment in {place_name}.'.format(place_name=place_name) %}
+{% else %}
+  {% set title = place_name + ' - '+ _('CHART_TITLE-CHART_CATEGORY-{category}'.format(category=category)) + ' - ' + _('Place Explorer') %}
+  {% set description = 'Statistics about {category} in {place_name}.'.format(category=category, place_name=place_name) %}
+{% endif %}
+{% set description = (place_summary or description_text) %}
 {% set subpage_title = _('Place Explorer') %}
 {% set subpage_url = url_for('place.place') %}
 {% set place_category = category %}
-{% set canonical_base_url = url_for('place.place', place_dcid=place_dcid) %}
-{% set canonical_category_url = url_for('place.place', place_dcid=place_dcid, category=category) %}
 
 {% block head %}
 <link rel="stylesheet" href={{url_for('static', filename='css/place_page.min.css' , t=config['VERSION'])}}>
 <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" />
-{% if category == '' %}
-<link rel="canonical" href="https://{{ request.host }}{{ canonical_base_url }}" />
-{% elif category != '' %}
-<link rel="canonical" href="https://{{ request.host }}{{ canonical_category_url }}" />
+
+{% if locale == "en" %}
+<meta name="description" content="{{ place_summary or description }}" />
```

Comment:
looks like some redundancy with the above `{% set description = (place_summary or description_text) %}`

Reply:
Good catch! Deduped.